README.md

```diff
@@ -1,73 +1,137 @@
 
-Key features of the application server:
- * 100% compatible drop-in replacement for Lua 5.1,
-   based on LuaJIT 2.1.
-   Simply use `#!/usr/bin/tarantool` instead of
-   `#!/usr/bin/lua` in your script.
- * full support for Lua modules and a rich set of
-   own modules, including cooperative multitasking,
-   non-blocking I/O, access to external databases, etc
+For more details, please see the
+[Tarantool documentation](https://www.tarantool.io/en/doc/2.2/).
 
-Key features of the database:
- * ANSI SQL, including views, joins, referential
-   and check constraints
- * MsgPack data format and MsgPack based
-   client-server protocol
- * two data engines: 100% in-memory with
-   optional persistence and an own implementation of LSM-tree, 
-   to use with large data sets
- * multiple index types: HASH, TREE, RTREE, BITSET
- * asynchronous master-master replication
- * authentication and access control
- * the database is just a C extension to the
-   application server and can be turned off
+## Getting started
 
-Supported platforms are Linux/x86 and FreeBSD/x86, Mac OS X.
+### Get hold of Tarantool
 
-Tarantool is ideal for data-enriched components of
-scalable Web architecture: queue servers, caches,
-stateful Web applications.
+* Try online:
 
-To download and install Tarantool as a binary package for your OS, please visit
-https://tarantool.io/en/download/.
+  Visit our playground at https://www.tarantool.io/en/try-dev/
 
-To build Tarantool from source, see detailed instructions in the Tarantool
-documentation at https://tarantool.io/en/doc/2.1/dev_guide/building_from_source/.
+* Use a Docker image:
 
-Please report bugs at https://github.com/tarantool/tarantool/issues
-We also warmly welcome your feedback in the discussion mailing
-list, tarantool@googlegroups.com.
+  ```bash
+  docker run -d -p 3301:3301 -v $(pwd):/var/lib/tarantool tarantool/tarantool:2
+  ```
 
-Thank you for your interest in Tarantool!
+* Install locally:
+
+  * On Ubuntu/Fedora/Debian/Centos:
+
+    ```bash
+    curl -L http://tarantool.io/installer.sh | VER=2.2 sudo bash
+    ```
+
+  * On Mac OS using [Homebrew](https://brew.sh):
+
+    ```bash
+    brew install tarantool
+    ```
+
+  For more details, please visit https://tarantool.io/download.
+
+* Build from source:
+
+  Please see
+  [detailed instructions](https://tarantool.io/doc/2.2/dev_guide/building_from_source/)
+  in the Tarantool documentation.
+
+### Connectors
+
+We have connectors for many popular languages:
+
+* [Python](https://github.com/igorcoding/asynctnt)
+* [PHP](https://github.com/tarantool-php/client)
+* [Java](http://github.com/tarantool/tarantool-java/)
+* [Go](https://github.com/tarantool/go-tarantool)
+
+[Check out connector details for other languages](https://www.tarantool.io/doc/2.2/book/connectors/)
+and connect to your current instance of Tarantool.
+
+## Use cases
+
+* Transactional cache
+* Persistent queues
+* ETL
+* Base for IoT platform
```

Comment:
Whether the list is like 'tarantool fit good for the following scenarious' or 'tarantool successfully used in the following projects'? I would say that the bullet 'Base for IoT platform' is about the latter, but the list looks as the former. But maybe I just don't know details. If so, can you explain? What makes IoT area being friendly to tarantool?

I would give @kostja's example: a social network, a database that stores 'like' counters for each post. Most counters are pretty cold (receive rare updates), but there are very popular users. When a new post is publised by such user, there are a lot of updates of one 'like' counter. This is where tarantool feels good, while a traditional database spends a lot of time on acquiring-releasing locks.